packages/manager/modules/exchange/src/account/export-as-csv/account-export-as-csv.controller.js

```diff
@@ -90,6 +90,8 @@ export default class ExchangeExportToCsvAccountsCtrl {
         'samaccountName',
         'taskPendingId',
         'id',
+        // https://answers.microsoft.com/fr-fr/msoffice/forum/all/excel-sortie-en-csv-avec-conversion-utf-8/0a05dc63-645d-4303-b0f5-c27513bf776e
```

Comment:
Link to an issue/solution written in english would be better 
Also why is this added above a property that doesn't seems to be transformed ?

Reply:
None of these properties are transformed, there are simply removed from the output for different reasons
The description field breaks the output file that's why it is removed. I'll find an english version of this issue :+1: